charts/enterprise-database/templates/database.yaml

```diff
@@ -0,0 +1,56 @@
+{{- $mongoDeploymentType := .Values.mongodb.type | default "ReplicaSet" -}}
+
+---
+apiVersion: mongodb.com/v1
+kind: MongoDB
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: {{ .Values.mongodb.type }}
+  members: {{ .Values.mongodb.members | default 3 }}
+  version: {{ .Values.mongodb.version | default "5.0.4" }}
```

Comment:
How can we maintain this default version?

Reply:
Maintain as in bumping this version whenever a new Mongodb version is released? That will have to be done manually for now.

Reply:
ok thanks